base/gmp.jl

```diff
@@ -547,6 +547,18 @@ end
 (-)(x::BigInt) = MPZ.neg(x)
 (~)(x::BigInt) = MPZ.com(x)
 
+# nand and nor
+nand(a::BigInt, b::BigInt) = MPZ.com(MPZ.and(a, b))
+nand(a::BigInt, b::BigInt, c::BigInt) = nand(nand(a, b), c)
```

Comment:
This is not what I would expect multi-arg `nand` to do. In all contexts I have encountered it, it was defined as `~(a & b & c)`. I think it would be better to just define the two-arg version for now though.

Reply:
Got it. I can remove the multi-arg versions for now.